### DISCUSSION TITLE
kodi播放不了视频了

### DISCUSSION THREAD

**Discussion by wanglei-github** [messense/aliyundrive-webdav#729] [General]
是kodi挂载，昨晚看得好好的，换了影片后，就打不开了，现在还是打不开，最新版1.11，大家有同样的问题吗

**Comment by OmEyOu** [comment]
我也是，刚还在看，现在也不行了，infuse也看不了，但是用小白云盘TV可以看

**Comment by wowoyang71** [comment]
挂了

**Comment by MznJeeHK** [comment]
昨天好好的，Emby今天能读出文件，但无法播放

**Comment by NeoZZZZ** [comment]
我也是，403错误

**Comment by robbienan** [comment]
我的也无法播放视频了

**Comment by gejun9558** [comment]
infuse昨晚还在看三体，今天早上就看不了了。更新了token也还是不行，文件能查看，但就是播放不了
